libs/libarchfpga/src/physical_types.h

```diff
@@ -1432,8 +1437,10 @@ struct t_segment_inf {
     float Rmetal;
     float Cmetal;
     enum e_directionality directionality;
+    enum e_parallel_axis parallel_axis;
```

Comment:
Comment what this data member means (other members are commented above; this one should be too).

Reply:
Added comments accordingly.